style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+}
+
+/* header block */
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 375px;
+  height: 92px;
+  background-color: #3e3c3c;
+  position: absolute;
+  left: 0;
+  top: 0;
+  box-sizing: border-box;
+}
```

Comment:
[OPTIONAL]:  It would be better to use a percentage instead of pixels, especially in the width of all your elements, that will make it easier for you when you want to make your project responsive . 😀👍✔